gitops/argocd/pvc-cleaner/kustomization.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+namespace: openshift-gitops
```

Comment:
Please revert this to the `pvc-cleaner` ns.

Reply:
Done